ConsistencyManagerTests/ListenerTests.swift

```diff
@@ -105,6 +105,7 @@ class ListenerTests: ConsistencyManagerTestCase {
                     listener = strongListener
 
                     addListener(strongListener, toConsistencyManager: consistencyManager)
+                    waitOnMainThread()
```

Comment:
not clear from this method call how long it waits. Is it possible to name this better?